tom_antares/antares.py

```diff
@@ -135,7 +114,7 @@ def alert_to_dict(cls, locus):
             'dec': locus.dec,
             'properties': locus.properties,
             'tags': locus.tags,
-            'lightcurve': locus.lightcurve.to_json(),
+            # 'lightcurve': locus.lightcurve.to_json(),
```

Comment:
Oops. I think I meant to ask about this one as well, thanks for catching it. My apologies that our API documentation for developers is... well, "lacking" is generous. It's in the works!

The `lightcurve` field is available from the API but it is lazy loaded from a different endpoint when the caller accesses it for the first time. Latency can be noticeable--more from Pandas constructing a dataframe than from the network requests--and I commented this line out to look at the improvement. It helps but I'm not sure if it's in the realm of unnecessary or not...

What's the plan for using the lightcurve data? Would it display on the query result page or on individual object pages?

Reply:
The idea is that photometric data is ingested into a TOM and associated with a target, so that an end user can keep track of photometry from all the various sources--observations, brokers, arbitrary data product uploads. At the moment, it's only implemented in MARS, and I'm not thrilled with the implementation, because it requires keeping some metadata about where the data is from in order to update it when more photometry comes in via the broker.

It's not necessary for this PR because really, it'd be new functionality, but I just wanted to make sure that it wouldn't be a problem in the future.